extensions/libraries/redcore/api/payment/helper.php

```diff
@@ -759,6 +763,26 @@ public static function updatePaymentStatus($paymentId)
 		return true;
 	}
 
+	/**
+	 * Check if ePay is using  instant capture
+	 *
+	 * @param   object	$payment	Payment object
+	 *
+	 * @return  boolean
+	 */
+	public static function isInstantCapture($payment)
+	{
+		$plugin 		= RApiPaymentHelper::getPaymentParams($payment->payment_name, $payment->extension_name, $payment->owner_name);
+		$instantcapture = false;
+
+		if ($plugin && strcmp($plugin->element, 'epay') === 0)
```

Comment:
maybe not setting it strictly to epay plugin... other payment plugins can have partial payment as well...

Reply:
We have:
   paypal
   epay
   bank transfer
   ean
   quickpay 

Can I please know to which ones this should be applied to?

I just know that qPay and paypal have. I dont think bank transfer have but I am not sure. I dont know what ean and quickspay are.

Reply:
better not to use a specific plugin name but some kind of flag. For instance, you can try to check some object plugin property value or plugin param like `if ($params->get('hasSomeSpecificAction', false))` ?

Reply:
@shandak Understood. I was told about that earlier too. 
But thing are quite messy and currently only ePay can handle capture.
You can see my other comments here.